src/carplay.cs

```diff
@@ -1008,4 +1289,556 @@ interface CPTemplateApplicationDashboardScene {
 		[Export ("dashboardWindow", ArgumentSemantic.Strong)]
 		UIWindow DashboardWindow { get; }
 	}
+
+	// warning BI1115: bgen: The parameter 'completionBlock' in the delegate 'CarPlay.CPListItemHandler' does not have a [CCallback] or [BlockCallback] attribute. Defaulting to [CCallback].
+	delegate void CPListItemHandler (CPBaseListItem item, Action completionBlock);
+
+	[Deprecated (PlatformName.iOS, 14, 0)]
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPBaseListItem : NSSecureCoding
+	{
+		[Deprecated (PlatformName.iOS, 14, 0)]
+		[NullAllowed, Export ("listItemHandler", ArgumentSemantic.Copy)]
+		CPListItemHandler ListItemHandler { get; set; }
+
+		[NullAllowed, Export ("userInfo", ArgumentSemantic.Strong)]
+		NSObject UserInfo { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPButton
+	{
+		[Export ("initWithImage:handler:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UIImage image, [NullAllowed] Action<CPButton> handler);
+
+		[NullAllowed, Export ("image", ArgumentSemantic.Copy)]
+		UIImage Image { get; }
+
+		[NullAllowed, Export ("title")]
+		string Title { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Field ("CPButtonMaximumImageSize")]
+		CGSize MaximumImageSize { get; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface CPContact : NSSecureCoding
+	{
+		[Export ("initWithName:image:")]
+		IntPtr Constructor (string name, UIImage image);
+
+		[Export ("name")]
+		string Name { get; set; }
+
+		[Export ("image", ArgumentSemantic.Strong)]
+		UIImage Image { get; set; }
+
+		[NullAllowed, Export ("actions", ArgumentSemantic.Copy)]
+		CPButton[] Actions { get; set; }
+
+		[NullAllowed, Export ("subtitle")]
+		string Subtitle { get; set; }
+
+		[NullAllowed, Export ("informativeText")]
+		string InformativeText { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (CPButton))]
```

Comment:
since `CPButton` disable `init` then this one too should
IOW add `[DisableDefaultCtor]`